engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -3850,6 +3908,9 @@ private Option add(Option opt)
     final Option exactMatch = add(new Option("exactMatch", "Only consider applications with exact app name"));
     final Option queue = add(OptionBuilder.withArgName("queue name").hasArg().withDescription("Specify the queue to launch the application").create("queue"));
     final Option force = add(new Option("force", "Force launch the application. Do not check for compatibility"));
+    final Option onlyConfigApps = add(new Option("onlyConfigApps", "Only use the apps present in the config package."));
+    final Option skipConfigApps = add(new Option("skipConfigApps", "Skip the apps present in the config package."));
+
```

Comment:
Instead of having two options that conflict each other, why don't we have one option that takes a value? for example --useConfigPackageApps={exclusive|inclusive|none}